index.html

```diff
@@ -0,0 +1,525 @@
+    </p><p>
+      Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+    </p><p>
+      So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+    </p><p>
+      We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+    </p><p>
+      “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+    </p>
+  </article>
+    
+  <aside class="sidebar">
+    <h3>Editors’ Picks</h3>
+    <hr>
+    <div class="sidebar-div">
+      <img src="https://static01.nyt.com/images/2020/07/14/magazine/14atwar-foroozan/14atwar-foroozan-square640.jpg?quality=75&auto=webp&disable=upscale&width=350" alt="picture1" 
+        class="sidebar-image"  >
+        <p>Boom Time for Death Planning</p>
+    </div>
+
+    <div class="sidebar-div">
+      <img src="https://static01.nyt.com/images/2020/07/11/travel/09travel-flight-promo/09travel-flight-promo-square640.jpg?quality=75&auto=webp&disable=upscale&width=350" alt="picture2"
+      class="sidebar-image">
+      <p>
+        How the Coronavirus Has Changed Air Travel: A Visual Diary of a Flight Continue reading the main story
+      </p>
+    </div>
+
+    <div class="sidebar-div">
+      <img src="https://tpc.googlesyndication.com/simgad/6075458894681406294?" alt="picture3"
+      class="sidebar-image">
+      <p>
+        <strong> PAID POST: Audemars Piguet</strong> Audemars Piguet’s Spiraling New Watch Museum Opens Its Doors
+      </p>
+    </div>
+  </aside>
+  <section>
+    <div class="article-footer">
+      <p>A version of this article appears in print on</p>
+      <p>March 18, 2014</p>
+      <p>, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a>Order Reprints</a> | <a>Today’s Paper</a> | <a>Subscribe</a></p>
+      <p>Icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a></p>
+      <br><br>
+      <div class="flex-container">
+        <div>
+          <button class="read-btn" type="button">READ 615 COMMENTS</button>
+        </div>
+        <div style="width:200px">
```

Comment:
- [ ] **MANDATORY** For good coding practices, don't use inline styling, please put this styles in your css file
![image](https://user-images.githubusercontent.com/53256139/87738096-3b324100-c7a2-11ea-977b-ede338d3191f.png)